storage/manager.go

```diff
@@ -109,8 +111,12 @@ func (m *Manager) AddPool(name string, pool engine.Engine, threshold int64) {
 	go pumper.Loop(m.PumpFn(name, pool, threshold))
 }
 
-func (m *Manager) AddJob(ctx context.Context, job *model.JobData) error {
-	return m.storage.BatchAddJobs(ctx, []*model.JobData{job})
+func (m *Manager) AddJob(ctx context.Context, name string, job *model.JobData) error {
+	err := m.storage.BatchAddJobs(ctx, []*model.JobData{job})
+	if err == nil {
+		metrics.storageAddJobs.WithLabelValues(name).Inc()
```

Comment:
I think we should use namespace/queue instead of the pool name

Reply:
Yes, just updated. Please have another look.